mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/ui/ProfileScreen.kt

```diff
@@ -0,0 +1,98 @@
+package org.mifos.mobilewallet.mifospay.home.ui
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.hilt.navigation.compose.hiltViewModel
+import org.mifos.mobilewallet.mifospay.R
+import org.mifos.mobilewallet.mifospay.designsystem.component.ProfileImage
+import org.mifos.mobilewallet.mifospay.home.presenter.ProfileViewModel
+import org.mifos.mobilewallet.mifospay.ui.utility.Orientation
+
+@Composable
+fun ProfileScreen(
+    viewModel: ProfileViewModel = hiltViewModel(),
+    onEditProfile: () -> Unit,
+    onSettings: () -> Unit,
+) {
+    val bitmap by viewModel.bitmapImage.collectAsState()
+
+    Column(modifier = Modifier.fillMaxSize()) {
+        ProfileImage(bitmap = bitmap)
+        Row(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(top = 12.dp, bottom = 20.dp),
+            horizontalArrangement = Arrangement.Center
+        ) {
+            Text(
+                text = "John Doe",
+                style = TextStyle(fontSize = 24.sp, fontWeight = FontWeight.Medium)
+            )
+        }
+
+        Column(
+            modifier = Modifier
+                .fillMaxSize()
+                .padding(start = 24.dp, end = 24.dp)
+        ) {
+            Row(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(bottom = 24.dp)
+            ) {
+                ItemCard(
+                    modifier = Modifier.weight(1f),
+                    icon = R.drawable.qrcode_black,
+                    text = R.string.personal_qr_code,
+                    onClick = {},
```

Comment:
I have left this blank deliberately as I wasnt sure what to implement in `Personal Qr Code`